src/assertions/OrderAssertions.ts

```diff
@@ -192,13 +192,26 @@ export class OrderAssertions {
     issuanceOrder: IssuanceOrder,
     fillQuantity: BigNumber,
   ) {
-    const { quantity } = issuanceOrder;
+    const {
+      quantity,
```

Comment:
are we still using all these variables?

Reply:
probably not, sorry just pushed out some changes and was trying to see if i could get the unit test working, but will get to this again later today